concepticondata/conceptlists/Duong-2020-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_GLOSS	CONCEPTICON_ID
+Duong-2020-100-1	1	sun	SUN	1343
+Duong-2020-100-2	2	moon	MOON	1313
+Duong-2020-100-3	3	star	STAR	1430
+Duong-2020-100-4	4	earth	EARTH (SOIL)	1228
+Duong-2020-100-5	5	cloud	CLOUD	1489
+Duong-2020-100-6	6	rain	RAINING OR RAIN	2108
```

Comment:
The list seems to be sorted by parts of speech, so we can be confident that this is RAIN (PRECIPITATION).

Reply:
I wasn't sure since there are no explicit indications, otherwise, but I can see your point.